.github/actions/setup-environment/action.yml

```diff
@@ -305,10 +308,12 @@ runs:
           if [ "$CRN_CREATE_CONTENTFUL_PR_ENV" = true ]; then
             echo "setting up dedicated PR env"
             echo "crn-algolia-index=asap-hub_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
+            echo "analytics-algolia-index=asap-hub-analytics_CI-$PR_NUMBER" >> $GITHUB_OUTPUT
```

Comment:
don't you think it is probably wise to add the app prefix (crn/gp2) already? I can imagine at some point the new requirement might be to introduce analytics to gp2